index.html

```diff
@@ -18,15 +18,130 @@
+        <div class="project project1">
+          <h2>Professional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+        <div class="project project2">
+          <h2>Data Dashboard Healthcare</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+        <div class="project project3">
+          <h2>Website Portfolio</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+        <div class="project project4">
+          <h2>Professional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+        <div class="project project5">
+          <h2>Data Dashboard Healthcare</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+        <div class="project project6">
+          <h2>Professional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a href="#" class="button">See Project</a>
+        </div>
+      </section>
+      <section id="about-me">
+        <div class="about-head">
+          <h2>About me</h2>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a href="#" class="button fs">Get My Resume</a>
```

Comment:
- [ ] Please the button "Get my resume" follows the button styling: **Active**, **Hover**, and **Pressed** 
- [ ] see design: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

-
![ho](https://user-images.githubusercontent.com/63972670/164566296-f85fe20d-8e93-41e8-b5f4-51321aa64e21.PNG)

Reply:
Hello, I corrected the project. Check it out, please. Best regards. 🙂